packages/vuetify/src/labs/VIconBtn/VIconBtn.tsx

```diff
@@ -161,6 +161,7 @@ export const VIconBtn = genericComponent<VIconBtnSlots>()({
 
       return (
         <props.tag
+          type={ props.tag === 'a' ? undefined : 'button' }
```

Comment:
tag could be router-link

```suggestion
          type={ props.tag === 'button' ? 'button' : undefined }
```